src/css/main.css

```diff
@@ -0,0 +1,223 @@
+box-shadow: 2px 2px 2px 0 rgba(0,0,0,0.2) inset;
+text-shadow: 1px 1px 0 rgba(255,255,255,0.66) ;
+-webkit-transition: all 200ms cubic-bezier(0.42, 0, 0.58, 1);
+-moz-transition: all 200ms cubic-bezier(0.42, 0, 0.58, 1);
+-o-transition: all 200ms cubic-bezier(0.42, 0, 0.58, 1);
+transition: all 200ms cubic-bezier(0.42, 0, 0.58, 1);
+-webkit-transform: none;
+transform: none;
+-webkit-transform-origin: 50% 50% 0;
+transform-origin: 50% 50% 0;
+}
+#fileUpload{
+	width: auto;
+}
+
+#IA{
+	color: #7b4397;
+}
+#create-index{
+  background: #7b4397;
+  background-image: -webkit-linear-gradient(top, #7b4397, #7b4397);
+  background-image: -moz-linear-gradient(top, #7b4397, #7b4397);
+  background-image: -ms-linear-gradient(top, #7b4397, #7b4397);
+  background-image: -o-linear-gradient(top, #7b4397, #7b4397);
+  background-image: linear-gradient(to bottom, #7b4397, #7b4397);
+  -webkit-border-radius: 0;
+  -moz-border-radius: 0;
+  border-radius: 0px;
+  text-shadow: 5px 0px 30px #666666;
+  -webkit-box-shadow: 5px 5px 10px #666666;
+  -moz-box-shadow: 5px 5px 10px #666666;
+  box-shadow: 5px 5px 10px #666666;
+  font-family: Arial;
+  color: #ffffff;
+  text-decoration: none;
+  width: auto;
+  margin-left: 10px;
+}
+
+#upload:hover{
+  background: #3cb0fd;
+  background-image: -webkit-linear-gradient(top, #3cb0fd, #3498db);
+  background-image: -moz-linear-gradient(top, #3cb0fd, #3498db);
+  background-image: -ms-linear-gradient(top, #3cb0fd, #3498db);
+  background-image: -o-linear-gradient(top, #3cb0fd, #3498db);
+  background-image: linear-gradient(to bottom, #3cb0fd, #3498db);
+  text-decoration: none;
+}
+
+#search-input {
+	float: left;
+    width: auto;
+    box-sizing: border-box;
+    border: 2px solid #ccc;
+    border-radius: 4px;
+    font-size: 16px;
+    background-color: white;
+    background-image: url('search-icon.png');
+    background-position: 10px 10px;
+    background-repeat: no-repeat;
+    padding: 12px 20px 12px 40px;
+    -webkit-transition: width 0.4s ease-in-out;
+    transition: width 0.4s ease-in-out;
+}
+#search-input:focus {
+    width: auto;
+}
+
+#search-expand{
+	width: auto;
+}
+#uploadDiv{
+    float:left;
+}
+.indexDiv{
+    max-height: 500px;
+    width: 100%;
+    overflow-y:scroll;
+    margin-top: 10px;
+}
+
+#filter-drop, #index-drop-list {
+    border: 1px solid #ccc;
+    width: 100px;
+    border-radius: 3px;
+    height: 40px;
+    overflow: hidden;
+    color:#7b4397;
+    background-color: white;
+}
+
+#filter-drop select, #index-drop-list select {
+    padding: 5px 8px;
+    width: 130%;
+    border: none;
+    box-shadow: none;
+    background: transparent;
+    background-image: none;
+    -webkit-appearance: none;
+    height: 40px;
+}
+
+#filter-drop select:focus, #index-drop-list select::focus {
+    outline: none;
+}
+#index-drop-list{
+    left:20px;
+    margin-left: 10px;
+}
+#showIndexId{
+    float: left;
+}
+#filter-drop{
+    margin-top: 5px;
+}
```

Comment:
- Format document for uniform indentation and spacing (I you wrote the entire document or added to it)